recipe/meta.yaml

```
@@ -49,7 +53,7 @@ test:
     - sysroot_linux-64 2.17  # [linux64]
     # make sure we pick up the version matching the docker,
     # or the linker would complain
-    - cudatoolkit 11.2
+    - cudatoolkit 11.2       # [cuda_compiler_version == "11.2"]
```

Think we need this to avoid solver issues

```suggestion
    - cudatoolkit 11.2|11.2.*       # [cuda_compiler_version == "11.2"]
```

---

Interesting that it works until I added a selector...